app/assets/stylesheets/footer.scss

```diff
@@ -0,0 +1,41 @@
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  background: #6e6a62 url('http://www.wediko.org/wp-content/themes/wediko/images/bg-footer.jpg');
```

Comment:
We should host this background locally rather than rely on a link we don't control.